extensions/wikidata/module/langs/translation-en_GB.json

```
@@ -95,17 +95,31 @@
     "perform-wikidata-edits/dialog-header": "Upload edits to Wikidata",
     "wikidata-account/connecting-to-wikidata": "Connecting to Wikidata…",
-    "wikidata-account/log-out": "Log out",
-    "wikidata-account/logged-in-as": "You are logged in as:",
-    "wikidata-account/log-in": "Log in",
-    "wikidata-account/close": "Close",
-    "wikidata-account/remember-credentials-label": "Remember credentials (stored unencrypted in OpenRefine's preferences)",
-    "wikidata-account/password-placeholder": "Enter your password",
-    "wikidata-account/password-label": "Password:",
-    "wikidata-account/username-placeholder": "Enter your username",
-    "wikidata-account/username-label": "Username:",
-    "wikidata-account/explain-log-in": "Logging in to <a href=\"https://www.wikidata.org/\" target=\"_blank\">Wikidata</a> lets you to upload edits directly from OpenRefine.",
     "wikidata-account/dialog-header": "Wikidata account",
+    "wikidata-account/explain-log-in": "Logging in to <a href=\"https://www.wikidata.org/\" target=\"_blank\">Wikidata</a> lets you to upload edits directly from OpenRefine.",
+    "wikidata-account/explain-owner-only-consumer-wiki": "See this <a href=\"https://github.com/OpenRefine/OpenRefine/wiki/Wikidata-owner-only-consumer\" target=\"_blank\">wiki</a> to get your owner-only consumer if you don't have one.",
+    "wikidata-account/explain-password-login": "You can also <a>login with your username/password.</a>",
+    "wikidata-account/explain-owner-only-consumer-login": "You can also <a>login with your owner-only consumer.</a>",
+    "wikidata-account/invalid-credentials": "Invalid credentials",
+    "wikidata-account/username-label": "Username:",
+    "wikidata-account/username-placeholder": "username",
+    "wikidata-account/password-label": "Password:",
+    "wikidata-account/password-placeholder": "password",
+    "wikidata-account/consumer-token-label": "Consumer token:",
+    "wikidata-account/consumer-token-placeholder": "consumer token",
+    "wikidata-account/consumer-secret-label": "Consumer secret:",
+    "wikidata-account/consumer-secret-placeholder": "consumer secret",
+    "wikidata-account/access-token-label": "Access token:",
+    "wikidata-account/access-token-placeholder": "access token",
+    "wikidata-account/access-secret-label": "Access secret:",
+    "wikidata-account/access-secret-placeholder": "access secret",
+    "wikidata-account/remember-me": "Remember me",
+    "wikidata-account/password-remember-me-title": "Your password won't be stored.",
+    "wikidata-account/owner-only-consumer-remember-me-title": "Consumer credentials are stored encrypted in cookies.",
+    "wikidata-account/close": "Close",
+    "wikidata-account/log-in": "Log in",
+    "wikidata-account/logged-in-as": "You are logged in as:",
```

Include the username parameter here so that the translators can use language specific ordering for the phrase:
```suggestion
    "wikidata-account/logged-in-as": "You are logged in as: $1",
```
You'll need to change the Javascript code with the $.i18n call too.